extension-manifest-v3/src/background/adblocker.js

```diff
@@ -232,18 +232,34 @@ ${scripts.join('\n\n')}}
   // wrapper to break the "isolated world" so that the patching operates
   // on the website, not on the content script's isolated environment.
   function codeRunningInContentScript(code) {
-    let content = decodeURIComponent(code);
     const script = document.createElement('script');
-    if (window.trustedTypes) {
-      const trustedTypePolicy = window.trustedTypes.createPolicy(
-        `ghostery-${Math.round(Math.random() * 1000000)}`,
-        {
-          createScript: (s) => s,
-        },
-      );
-      content = trustedTypePolicy.createScript(content);
+    script.async = false;
+
+    let scriptlets = decodeURIComponent(code);
+
+    if (navigator.userAgent.includes('Firefox')) {
```

Comment:
I think this is place to use `__PLATFORM__`

```suggestion
    if (__PLATFORM__ === 'firefox') {
```